visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-dialog-content class="content">
+	<div class="row">
+		<div class="title">
+			<h3>Custom Views</h3>
+		</div>
+		<div class="action-buttons">
+			<cc-upload-custom-config-button></cc-upload-custom-config-button>
+			<cc-download-custom-configs-button></cc-download-custom-configs-button>
+			<cc-add-custom-config-button class="custom-configs-button-in-custom-views"></cc-add-custom-config-button>
+		</div>
+	</div>
+	<mat-divider></mat-divider>
+	<div
+		class="no-custom-configs-box"
+		*ngIf="dropDownCustomConfigItemGroups.applicableItems.size === 0 && dropDownCustomConfigItemGroups.nonApplicableItems.size === 0"
+	>
+		<span>It is time to add your first Custom View!</span>
+	</div>
+	<mat-accordion class="custom-config-container" [multi]="true">
+		<cc-custom-config-item-group
+			[customConfigItemGroups]="dropDownCustomConfigItemGroups.applicableItems"
+		></cc-custom-config-item-group>
+		<cc-custom-config-item-group
+			*ngIf="!isCollapsed"
+			class="non-applicable-group"
```

Comment:
[Optional] 
```suggestion
			class="non-applicable-config-item-group"
```
Don't forget to update class name in style as well if applied